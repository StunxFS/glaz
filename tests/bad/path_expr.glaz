fn a() {}

fn main() {
  a()::s1;
}
