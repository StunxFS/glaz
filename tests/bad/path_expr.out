tests/bad/path_expr.glaz:4:4: error: bad use of path expression
     | 
   4 |   a()::s1;
     |   ^^^^^^^
glaz error: could not compile `path_expr`, aborting due to previous error.
